tools/ReadConsoleInputStream/ReadConsoleInputStreamDemo.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <RootNamespace>Nivot.Terminal</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Pipelines.Sockets.Unofficial" Version="2.0.22" />
+    <PackageReference Include="Vanara.PInvoke.Kernel32" Version="2.3.6" />
```

Comment:
>"Vanara.PInvoke.Kernel32 [](start = 30, length = 24)

Well that's nifty.

Reply:
Yeah -- I'm tired of rewriting NativeMethods classes or pulling signatures off of pinvoke.net and not having them work. I like his API design. There's another popular nuget package that's far more literal and uses a lot of unsafe code. His does not.